### ISSUE TITLE
Notify users of Content (Question or Options) changes

### ISSUE THREAD

**Issue by serenalomonico** [shcaba/FishPath-updates#129]
Users get an email from "FishPath Admin" that a new question or a new option was added.
OR, each time a user logs in, a pop up or flag is presented where the user has to fill out any new questions that were added, and must answer them before proceeding.

**Comment by serenalomonico** [comment]
Separate but related flag or note for "Power Users" (those who are running workshops, for ex) about if new assessments have been added, or other changes in greater detail

**Comment by bsnouffer** [comment]
I agree that a formal process for notifying users of changes should be put in place. I noticed this when updating answers to time series questions in #168. The answers were edited instead of removed and new ones added to keep the criteria and caveat associations in place. However, users answers might no longer match the new wording of the questions. They will most likely not notice this silent change.

**Comment by dtdougherty** [comment]
Brian- can you describe this in more detail? Why would answers no longer
match the new wording, what is an example?

**Comment by bsnouffer** [comment]
In this case the old answers had the term 'usable' in the answers which was agreed to be ambiguous and users answering the survey might interpret it differently. So a user might have answered a time series question with 10+ 'usable' years thinking that they had 10 years of data so it's "usable". But now the answers clarify what it means to be "usable". 

2: Time series with flaws such as missing years of major or peak catch, missing major fleets/metiers or is generally poorly sampled (i.e., high uncertainty).
3: Partial time series that includes major/peak years of catches from major fleets/metiers.

The data that was described as "10+ 'usable' years" might in fact be a time series missing major fleets or have other flaws. So the answer in this case should really be "2: time series with flaws...", when its going to show up as "3: partial time series that includes...". 

Same could be true of the reverse situation. A fishery might have 5 years of quality data for a time series. So in the old answers they select "2: 5-10 'usable' years. when under the new wording it might fit better under "3: partial time series that includes..."

Hope this helps clarify.

**Comment by dtdougherty** [comment]
Got it, thank you. This is something we all need to consider for all of our
previous runs that we are still referencing and reporting from. Even our
Peru answers from last week.
